identity/middleware.go

```diff
@@ -11,14 +11,114 @@
 //    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
+
 package identity
 
 import (
+	"net/http"
+	"regexp"
+
 	"github.com/ant0ine/go-json-rest/rest"
+	"github.com/gin-gonic/gin"
 
 	"github.com/mendersoftware/go-lib-micro/log"
+	urest "github.com/mendersoftware/go-lib-micro/rest.utils"
 )
 
+const (
+	// IdentityPathsRe decides which requests
+	defaultPathRegex = "^/api/management/v[0-9.]{1,6}/.+"
```

Comment:
why `{1,6}` and not simply `+`?

Reply:
Fair enough. I just thought of caputring a semver and nothing else.